src/client-functions/selector-builder/add-api.js

```diff
@@ -88,8 +90,23 @@ async function getSnapshot (getSelector, callsite) {
     return node;
 }
 
-function addSnapshotPropertyShorthands (obj, getSelector) {
-    SNAPSHOT_PROPERTIES.forEach(prop => {
+function assertSnapshotExtensionOptions (extensions) {
```

Comment:
I've discussed it with @VasilyStrelyaev, he didn't see `addCustomDOMProperties` variant. Let's stick with it. Also, make sure that naming is consistent across codebase